### PR TITLE
Bug - link not showing

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Fixes meeting link not showing up in the location field on emails and calendar

Fixes # (issue)
#2332 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Create an email with a link type
- [ ] Link will now show in confrimation emails/calendar

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by alannnc — APPROVED** [review]
Code looks good but haven’t tested it

**Review by alannnc — COMMENTED** [review]
I updated the PR, as to me link was missing on organiser and attendee emails.
Take a look at my fix and if it doesn't break anything else.
@sean-brydon 
https://github.com/calcom/cal.com/pull/2333/commits/e3c1cd5134e99510f071462c7ddaed6954297cd0

File: apps/web/lib/emails/templates/organizer-scheduled-email.ts [hunk 1]
```diff
@@ -307,6 +307,14 @@ ${getRichDescription(this.calEvent)}
       providerName = location[0].toUpperCase() + location.slice(1);
     }
 
+    // If location its a url, probably we should be validating it with a custom library
+    if (
+      this.calEvent.location &&
+      (this.calEvent.location.includes("https://") || this.calEvent.location.includes("http://"))
```
Comment (zomars) [inline apps/web/lib/emails/templates/organizer-scheduled-email.ts:313]: This will return true for URLs like: `notanUrlButHashttps://intheMiddle`

**Review by zomars — APPROVED** [review]
Fixed some minor details, nice work @sean-brydon 🙏🏽

**Pull request merged**